internal/provider/serviceUtils.go

```diff
@@ -453,11 +453,11 @@ func setDataToService(d *schema.ResourceData, diags diag.Diagnostics, srv IServi
 	if d.HasChange("client_id_alias_enabled") {
 		srv.SetClientIdAliasEnabled(d.Get("client_id_alias_enabled").(bool))
 	}
-	if d.HasChange("attributes") {
+	if d.HasChange("attribute") {
```

Comment:
I think we should keep this as "attributes" and update the documentation to reflect that.

Both in our OpenAPI docs and Java library the property name is "attributes".